src/main/java/perficient/academic/spring5webapp/model/Course.java

```diff
@@ -0,0 +1,25 @@
+package perficient.academic.spring5webapp.model;
+
+
+import jakarta.persistence.*;
```

Comment:
We can check the imports here, use * is a bad practice, tomorrow we can check you IDLE configuration